presto-docs/src/main/sphinx/release/release-346.md

```diff
@@ -0,0 +1,60 @@
+* Add SQL syntax for `GRANT` and `REVOKE` on schema. This is not yet used by any connector. ({issue}`4396`)
+* Add SQL syntax for `ALTER TABLE SET AUTHORIZATION` to change the table owner. ({issue}`5717`)
+* Fix incorrect handling of negative offsets for the `time with time zone` type. ({issue}`5696`)
+* Fix incorrect result when casting `time(p)` to `timestamp(p)` for precisions higher than 6. ({issue}`5736`)
+* Fix improper table alias visibility for queries that select all fields. ({issue}`5660`)
+* Fix failure when query parameter appears in a lambda expression. ({issue}`5640`)
+* Fix failure for queries containing `DISTINCT *` and fully-qualified column names in the `ORDER BY` clause. ({issue}`5647`)
+
+## Web UI Changes
+
+* Fix "Capture Snapshot" button on the Worker page. ({issue}`5759`)
+
+## JDBC Driver Changes
+
+* Support number accessor methods like `ResultSet.getLong` or`ResultSet.getDouble`
+  on `decimal` values, as well as `char` or `varchar` that can be unambiguously interpreted as numbers. ({issue}`5509`)
+* Add `SSLVerification` JDBC connection parameter that allows to skip SSL verification on `SSLVerification=NONE` or to skip SSL 
+  host name verification on `SSLVerification=CA`. ({issue}`5610`)
+* Remove legacy `useSessionTimeZone` JDBC connection parameter. ({issue}`4521`)
+  
+## BigQuery Connector Changes
+
+* Fix issue when query could return invalid results, if some column references were pruned out during query optimisation. ({issue}`5618`)
+
+## Cassandra Connector Changes
+
+* Improve performance of `INSERT` queries with batch statement. The batch size can be configured via `cassandra.batch-size`
+  configuration property. ({issue}`5047`)
+
+## Hive Connector Changes
+
+* Support coercion between `varchar` types of different lengths. ({issue}`5484`)
+* Improve query performance by adding support for dynamic bucket pruning. ({issue}`5634`)
+* Add support for `ALTER TABLE SET AUTHORIZATION` SQL syntax to change the table owner. ({issue}`5717`)
+* Remove deprecated `parquet.fail-on-corrupted-statistics` (previously known as `hive.parquet.fail-on-corrupted-statistics`).
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+
+## Iceberg Connector Changes
+
+* Remove deprecated `parquet.fail-on-corrupted-statistics` (previously known as `hive.parquet.fail-on-corrupted-statistics`).
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+    
+## Kudu Connector Changes
+
+* Improve performance of queries having only `LIMIT` clause. ({issue}`3691`)
+
+## Oracle Connector Changes
+
+* Add support for setting column comments. ({issue}`5399`)
+* Allow enabling remarks reporting via `oracle.remarks-reporting.enabled ` config property. ({issue}`5720`)
```

Comment:
Is it obvious that "remarks reporting" refers to "column comments"?

Reply:
Not to me. I had no idea they were related.

@losipiuk ?